README

```diff
@@ -1,3 +1,30 @@
+*** Description ***
+
+The wolfSSL embedded SSL library (formerly CyaSSL) is a lightweight SSL/TLS
+library written in ANSI C and targeted for embedded, RTOS, and
+resource-constrained environments - primarily because of its small size, speed,
+and feature set.  It is commonly used in standard operating environments as well
+because of its royalty-free pricing and excellent cross platform support.  wolfSSL
+supports industry standards up to the current TLS 1.2 and DTLS 1.2 levels, is up
+to 20 times smaller than OpenSSL, and offers progressive ciphers such as ChaCha20,
+Curve25519, NTRU, and Blake2b.  User benchmarking and feedback reports
+dramatically better performance when using wolfSSL over OpenSSL.
+
+wolfSSL is powered by the wolfCrypt library. A version of the wolfCrypt
+cryptography library has been FIPS 140-2 validated (Certificate #2425). For
+additional information, visit the wolfCrypt FIPS FAQ
+(https://www.wolfssl.com/license/fips/) or contact fips@wolfssl.com
+
+*** Why choose wolfSSL? ***
+
+There are many reasons to choose wolfSSL as your embedded SSL solution. Some of
+the top reasons include size (typical footprint sizes range from 20-100 kB),
+support for the newest standards (SSL 3.0, TLS 1.0, TLS 1.1, TLS 1.2, TLS 1.3,
+DTLS 1.0, and DTLS 1.2), current and progressive cipher support (including stream
+ciphers), multi-platform, royalty free, and an OpenSSL compatibility API to ease
+porting into existing applications which have previously used the OpenSSL package.
+For a complete feature list, see https://github.com/wolfSSL/wolfssl/wiki/Features.
```

Comment:
This should link to the wolfSSL product page on our main website, instead of the GitHub wiki.